### PR TITLE
libreoffice deps

### PR DESCRIPTION
**Pull request by knyipab**
Related #1174

### CODE REVIEW

**Comment by licy183** [comment]
Emmm... Do they all have to be explicitly linked to `compiler-rt` statically?

**Comment by knyipab** [comment]
Do you mean the `$($CC -print-libgcc-file-name)` patch? Not all of them but many of them prompt the relevant errors in GitHub actions builds, and I added that only when needed. It seems that all `librevenge`-dependent packages would need that patch.

File: tur/box2d/0002-unsupported-cpu.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/src/core.h b/src/core.h
+index fe495c6..baaa5b3 100644
+--- a/src/core.h
++++ b/src/core.h
+@@ -47,7 +47,7 @@
+ #elif defined( __EMSCRIPTEN__ )
+ 	#define B2_CPU_WASM
+ #else
+-	#error Unsupported CPU
```
Comment (licy183) [inline tur/box2d/0002-unsupported-cpu.patch]: This is a little strange... I'll have a test this weekend.

**Comment by licy183** [comment]
Thanks for working on this!

**Pull request merged**